### ISSUE TITLE
`exp` doesn't have `toTex()` defined

### ISSUE THREAD

**Issue by mattvague** [josdejong/mathjs#2583]
E.g.

<img width="218" alt="image" src="https://user-images.githubusercontent.com/64985/171308739-c7f77da9-2b6d-4a8a-b18d-72525007e9aa.png">

**Comment by josdejong** [comment]
Hm, yeah, I think that makes sense. It is fine with me to change this. This is defined here: https://github.com/josdejong/mathjs/blob/develop/src/utils/latex.js#L123

Do you want to create a PR fixing this?

**Comment by LloydLS** [comment]
Hi, 
Would it be possible to make this controllable in some way? 

For exponentials containing a division, I personally find the `e^()` format to be less readable because the exponent fraction is too small (I think it's a single line height `\tfrac{}{}`).

e.g. for `parse('exp(-E / RT)').toTex()`, I have a preference for the result `\exp\left(\frac{- E}{ RT}\right)` instead of `e^{\left(\frac{- E}{ RT}\right)}`

**Comment by josdejong** [comment]
Thanks for your inputs. 

If there are different preferences per user and per case, it will be hard to make everyone happy. Can we come up with a clear and straightforward list of cases where we would want either one or the other?

The current implementation already allows perfect control: write either `exp(...)` or `e^(...)` depending on how you want the output to look. And with a transform of the parsed expression you could already replace `exp(...)` with `e^(...)` if you want so.

**Comment by LloydLS** [comment]
Hi,

> If there are different preferences per user and per case, it will be hard to make everyone happy. 
Yep, totally agree.

> Can we come up with a clear and straightforward list of cases where we would want either one or the other?
Probably, but probably unlikely to satisfy everyone.

> The current implementation already allows perfect control: write either exp(...) or e^(...) depending on how you want the output to look. And with a transform of the parsed expression you could already replace exp(...) with e^(...) if you want so.
Great! Then I suppose I like the current approach better.

Best regards

**Comment by josdejong** [comment]
Yes maybe it's the easiest to keep the current behavior like it is. What do you think @mattvague?

**Comment by mattvague** [comment]
What if we just added config options for this kind of thing? Maxima does something similar: https://maxima.sourceforge.io/docs/manual/maxima_9.html#exptdispflag

**Comment by josdejong** [comment]
👍
